mlflow/R/mlflow/R/model-keras.R

```diff
@@ -0,0 +1,55 @@
+#' importFrom mlflow create_conda_env, create_pyfunc_conf
+#' @export
+mlflow_save_flavor.keras.engine.training.Model <- function(x,
+                                                           path = "model",
+                                                           r_dependencies=NULL,
+                                                           conda_env=NULL) {
+  save_model_hdf5(x, filepath = file.path(path, "model.h5"), include_optimizer = TRUE)
```

Comment:
So, ideally, the `keras` package implements these methods; however, I think is a good approach to add them first here since coordinating package releases can be tricky.

The only challenge we have here is that we can't use any `keras` functions, directly. We need to load them all dinamically as follows:

```r
if (!"package:keras" %in% search()) {
  stop("The 'keras' package is not available, use 'library(keras)' before exporting a model.")
}

save_model_hdf5 <- get("save_model_hdf5", envir = as.environment("package:keras"))
```

Then you can use `save_model_hdf5` as expected.